hikari/impl/interaction_server.py

```diff
@@ -189,7 +205,10 @@ def is_alive(self) -> bool:
         """
         return self._server is not None
 
-    async def _fetch_public_key(self) -> ed25519.VerifierT:
+    async def _fetch_public_key(self) -> signing.VerifyKey:
+        # This is kept inline as pynacl is an optional dependency.
+        from nacl import signing
```

Comment:
instead of re-importing this i'd be safer to just assign signing to an attribute in the init